observer.go

```diff
@@ -25,19 +25,17 @@ type Observer struct {
 }
 
 var (
-	// Metric for node active status
-	nodeActiveDesc = prometheus.NewDesc(
-		"aerospike_node_up",
-		"Aerospike node active status",
-		[]string{"cluster_name", "service", "build"},
-		nil,
-	)
+	// aerospike_node_up metric descriptor
+	nodeActiveDesc *prometheus.Desc
 
 	// Node service endpoint, cluster name and build version
 	gService, gClusterName, gBuild string
 
 	// Number of retries on info request
 	retryCount = 3
+
+	// Default info commands (order is important here)
+	defaultInfoKeys = []string{"cluster-name", "service-clear-std", "build"}
```

Comment:
At first, I thought this is a good step in the right direction by removing magic strings from the files, but after I finished reading the code, it dawned on me that `defaultInfoKeys[1]` is much less readable than `"service-clear-std"`.
Maybe the better approach would be to have them as vars:
```go
var (
    ikClusterName = "cluster-name"
    ikService          = "service-clear-std"
    ikBuild              = "build"
)
```
This way we gain all the benefits without any of the readability problems. You can also change the content whenever you want.

Reply:
Right, makes sense. Done.